internal/server/util/jwt_service.go

```diff
@@ -0,0 +1,208 @@
+package util
```

Comment:
Not sure whether `jwt_service.go` is a good name for this: a `_service.go` in Koko generally represents an grpc/wrpc/rest API service, while this is some more of an util service. What do you think?

Reply:
Done